MIGRATION.md

```diff
@@ -252,7 +259,6 @@ favor of string literals.
 
 ### Removed Enums
 * The previously deprecated enum `Status` was removed (see [#4891](https://github.com/getsentry/sentry-javascript/pull/4891)).
-  [This example](#status) explains how to migrate.
```

Comment:
Was this deletion intentional?

Reply:
This was intentional as the example no longer applies. We removed `eventStatusFromHttpCode` in https://github.com/getsentry/sentry-javascript/pull/5002.

Reply:
I added a line that we removed that function in 9536281

Reply:
Got it, thanks for clearing that up